src/Schema/Events/FilterAndSort.ts

```diff
@@ -63,3 +82,31 @@ export interface CommercialFilterParamsChanged {
   current: CommercialFilterParams
   changed: CommercialFilterParams
 }
+
+/**
+ * A user applies filters to a filterable/sortable auction results module
```

Comment:
Do we definitely want a separate event from `CommercialFilterParamsChanged`? The existing one could handle this by making `current`/`changed` accept `AuctionResultsFilterParams | CommercialFilterParams`.

Reply:
That's a good point. One event, separate params. I did this mainly because right now on web it _is_ a separately named event altogether, but we can certainly go with a single event.

Reply:
Like this? 
![Screen Shot 2020-11-23 at 1 11 35 PM](https://user-images.githubusercontent.com/26879583/99999103-a9f55b80-2d8d-11eb-9a3d-4b6bff5d905e.png)

Reply:
And then we subsequently remove the references to `auctionResultsFilterParamsChanged` throughout